features/app-launch-handler.yml

```
@@ -0,0 +1,9 @@
+name: Launch Handler
+description: The `launch_handler` manifest member is used to configure how web app launches should behave with respect to already open web app instances.
```

Adding "web app" for consistency with other features like this.

Reading the spec, it sounds like the concept for "launch handlers" might expand in the future. Perhaps we can be more explicit here about which part of the feature we're talking about, so if this ever changes we'll know to add a second feature:

```suggestion
description: The `launch_handler` and `client_mode` web app manifest members configure how web app launches should behave when an instance of the app is already open. For example, you can choose to focus on an existing app instead of launching a new one.
```